### ISSUE TITLE
Draft posts appearing in RSS

### ISSUE THREAD

**Issue by Raemon** [LessWrong2/Lesswrong2#1977]
From intercom:

"Just FYI, I got [title redacted] pushed to my RSS Feed at 10:54PM UTC. I wasn't able to view any comments, but I did see the maintext discussing that the post would be opened to public viewing later. When I attempted to view the comments, I received the "Sorry, we couldn't find what you were looking for." message. I don't know if this is the behaviour you expected."

I think what probably happened is that I (Raemon) accidentally published a draft, and then un-published it. This doesn't seem like the worst behavior but I think it might make sense for all RSS feeds to require at least one upvote on a post before it goes out, or something.

**Comment by jimrandomh** [comment]
Yes, I believe that's what happened (post was published, then moved back to drafts). The way feed-readers like Feedly work, once a post has been in the RSS feed, we can't take it back or edit it. We could somewhat mitigate this by introducing a time delay or upvote requirement, but this is basically working as designed.
